tasks/update-cmr-access-constraints/schemas/output.json

```diff
@@ -0,0 +1,42 @@
+{
+  "title": "PostToCmrOutput",
```

Comment:
Should update the title and description here

Reply:
fb86abf 👍